server.js

```diff
@@ -1,34 +1,34 @@
 // TASK 1: Setting up server
 
 
-const http = require('http');
+// const http = require('http');
 
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
-  res.end('Hello World\n');
-});
+// const server = http.createServer((req, res) => {
+//   res.statusCode = 200;

```

Comment:
Duplicate declaration of constant 'PORT'. Consider renaming one of them to avoid confusion.